util/llpcElfWriter.cpp

```diff
@@ -955,6 +966,117 @@ void ElfWriter<Elf>::MergeElfBinary(
     WriteToBuffer(pPipelineElf);
 }
 
+template<class Elf>
+void ElfWriter<Elf>::Reinitialize() {
+    m_map.clear();
+    m_notes.clear();
+    m_symbols.clear();
+
+    m_sections.push_back({});
+
+    m_strtabSecIdx = m_sections.size();
+    m_sections.push_back({});
+    m_map[StrTabName] = m_strtabSecIdx;
+
+    m_textSecIdx = m_sections.size();
+    m_sections.push_back({});
+    m_map[TextName] = m_textSecIdx;
+
+    m_symSecIdx = m_sections.size();
+    m_sections.push_back({});
+    m_map[SymTabName] = m_symSecIdx;
+
+    m_noteSecIdx = m_sections.size();
+    m_sections.push_back({});
+    m_map[NoteName] = m_noteSecIdx;
+}
+
+template<class Elf>
+Result ElfWriter<Elf>::LinkRelocatableElf(const llvm::ArrayRef<ElfReader<Elf>*>& relocatableElf, Context* pContext) {
```

Comment:
`Result ElfWriter<Elf>::LinkRelocatableElf(const llvm::ArrayRef<ElfReader<Elf>*>& relocatableElf, Context* pContext) {`
->
```
Result ElfWriter<Elf>::LinkRelocatableElf(
    const ArrayRef<ElfReader<Elf>*>& relocatableElfs, // [in] ...
    Context*                         pContext)        // [in] ...
```
Don't add "llvm" prefix to .cpp source file.
relocatableElf -> relocatableElfs, it is an array, highlight we have multiple ELFs